plugins/BEdita/Core/tests/TestCase/Utility/ObjectsHandlerTest.php

```diff
@@ -56,6 +57,13 @@ public function setUp()
      */
     public function tearDown()
     {
+        // seems not necessary, but without `fixtureManager->shutDown`
+        // an integrity constraint error on foreign key is raised on SQLitr
+        $info = Database::basicInfo();
+        if ($info['vendor'] === 'sqlite') {
+            $this->fixtureManager->shutDown();
```

Comment:
If it's necessary, then I'd simply trigger the `shutDown()` for all database vendors, with an explanatory comment — it would be invoked for all vendors, but it's a test, so we don't care.

I'd like to investigate it a bit, though…

Reply:
Problem is: with MySQL and postgres invoking `shutDown()` here causes a failure....nice, isn't it :smiley: ?

I tried to find out why, without success: I think this has to do with #1178 #1371

Launching this unit test alone is fine, problem is the test sequence.

Reply:
I tried merging the source branch of PR #1371 into this source branch, and the problem seems to have vanished… 🤔 

We might leave a `// TODO` comment to remind us that this part could be removed if/when both PRs are merged.

Reply:
Better: we can merge #1371 and then remove `shutDown()`